gradle/libs.versions.toml

```diff
@@ -87,7 +87,7 @@ squareup-retrofit = '2.11.0'
 violation-comments = '1.70.0'
 wiremock = '2.26.3'
 wordpress-aztec = 'v2.1.4'
-wordpress-fluxc = '2.97.0'
+wordpress-fluxc = '2.98.0'
```

Comment:
Minor (🔍): Why not using the latest [2.99.0](https://github.com/wordpress-mobile/WordPress-FluxC-Android/releases/tag/2.99.0) instead? 🤔 

PS: I understand that [2.99.0](https://github.com/wordpress-mobile/WordPress-FluxC-Android/releases/tag/2.99.0) doesn't have any changes on top of [2.98.0](https://github.com/wordpress-mobile/WordPress-FluxC-Android/releases/tag/2.98.0), but still, it is the latest, just to avoid confusion, please ignore if you feel it is irrelevant anyway... 🤷

Reply:
Actually, I double checked and 2.99.0 has some changes comparing to 2.98.0... https://github.com/wordpress-mobile/WordPress-FluxC-Android/compare/2.98.0...2.99.0 (it doesn't look like from reading the release changelog, right?)

I'll update to 2.99.0, thanks!